pwdtk/middlewares.py

```diff
@@ -84,12 +85,25 @@ def process_exception(self, request, exception):
 
     def process_request(self, request):
 
-        if self.must_renew_password(request):
+        if self.must_renew_password(request, response=False):
```

Comment:
if response is normally a response object, then the default should be `None` and not `False`

if it is really a boolean, then perhaps better to chose a name like
`is_response` `has_response` or `response_flag`

Reply:
refactored ;)